Makefile

```diff
@@ -23,43 +23,60 @@ export file
 export DOCKER_BUILDKIT=1
 export COMPOSE_DOCKER_CLI_BUILD=1
 
+ci-docker-compose := docker-compose -f base.yml  -f .ci/docker-compose.yml
+
+##
+##+------------------------------------------------------------------------+
+##| Help                                                                   |
+##+------------------------------------------------------------------------+
 help: ## Show this help.
 	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/ ##/			/' | sed -e 's/##//'
 
+##
+##+------------------------------------------------------------------------+
+##| Development                                                            |
+##+------------------------------------------------------------------------+
+
 check: ## Check the code style using black, mypy and pylint.
 	make black
 	make mypy
 	make pylint
 
 mypy: ## Check code style using mypy.
-	docker-compose -f base.yml -f .ci/docker-compose.yml \
-		run --rm mula \
+	$(ci-docker-compose) run --rm mula \
```

Comment:
What a beautiful idea for a cleanup ;)

Reply:
Haha, and ofcourse many thanks or that! :D